tests/src/test_app.py

```diff
@@ -711,3 +711,16 @@ def test_attribute_filter_rawdata() :
     print(query_result)
     assert query_result.encode('utf-8') == expected_query.encode('utf-8')
 
+def test_osm_recency_query():
```

Comment:
Tests are good , since query is short and obvious I will suggest to have test with the  different parameter combination , for example with paramer saying insight as source and rawdata as data output , things like that , to check the validation as well